deployment/certs/ca-cert-policy-curl.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "policy": {
```

Comment:
This file content can be placed inside `ca-cert-policy-curl.json`

Reply:
Not really, `az rest` requires a slightly different JSON format. If you try creating a new ca file using `ca-cert-policy-curl.json`, you will encounter an error.